akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -129,12 +129,11 @@ object ActorMaterializer {
    * INTERNAL API
    */
   private[akka] def downcast(materializer: Materializer): ActorMaterializer =
-    materializer match {
+    materializer match { //FIXME this method is going to cause trouble for other Materializer implementations
```

Comment:
From ActorMaterializer:

~~~scala
  /**
   * INTERNAL API: this might become public later
   */
  private[akka] def actorOf(context: MaterializationContext, props: Props): ActorRef

  /**
   * INTERNAL API
   */
  private[akka] def system: ActorSystem

  /** INTERNAL API */
  private[akka] def supervisor: ActorRef
~~~